microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -290,6 +304,11 @@ def _kdump_test(self, node: Node, log: Logger) -> None:
                     f"{identifier.__class__.__name__}: {identifier}. Retry..."
                 )
                 time.sleep(10)
+            if not stdout:
+                serial_console = node.features[SerialConsole]
+                serial_console.check_rootfilesystem_exception(
```

Comment:
It may generate a lot of log. It needs to reduce log. For example, check the log every 1 minute.

Reply:
Actually, it doesn't generate a lot of logs. 
If the VM can be connected, the 'stdout' should not be NULL. So it won't download the console log.
If the VM can't be connected in 300s (the timeout of 'try_connect' is 300s), it will download the console log. If there is initramfs, it will raise exception. If not, it will keep to try_connect. The kdump timeout is 800s, so it will download the console log 3 times at most.